src/plugins/widgets/search/Search.tsx

```diff
@@ -23,24 +28,126 @@ class Search extends React.PureComponent<Props & InjectedIntlProps, State> {
   static defaultProps = {
     engine: 'google',
     placeholder: '',
+    suggestions: {
```

Comment:
I think you accidentally left these settings inside the `suggestion` object (you moved them out and specified your reason why in the interface). Maybe we could have "suggestion" incorporated into the setting name, say `suggestionsActive` and `suggestionsQuantity`?